src/index.html

```diff
@@ -22,6 +22,316 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
```

Comment:
Ensure that the image path './images/logo.png' is correct and that the image exists in the specified directory. This will prevent broken image links.